aces/Loris
# [Instrument] Rename Data_entry_completion_status  & move to its own column in flag

## Brief summary of changes
This PR moves Data_entry_completion_status field to its own column in flag following discussion and renames to Required_elements_completed. This is accomplished with the help of a patch and a single-use script (the patch must be run before the script).

#### Testing instructions (if applicable)

1. Run patches `SQL/New_patches/2020-07-29_Required_elements_completed_flag_column.sql` and `SQL/Cleanup_patches/2020-12-08-remove_dataEntryCompletionStatus_conflict_resolver.sql`
2. Run `php tools/single_use/Set_Required_elements_completed_flag.php`. Check that the script successfully sets the new column to Y for instruments that already have Data_entry_completion_status=Complete and to N for instruments that had Data_entry_completion_status=Incomplete. Make sure that this holds for both JSON and non JSON instruments.
3. Run `php tools/single_use/Remove_Data_entry_completion_status_instr_column.php`
4. Check that the Data column of flag no longer contains any Data_entry_completion_status values.
5. Check in the backend that Data_entry_completion_status is only set to "Complete" when all required elements have been filled & saved
6. Check on the front end that the functionality for being able to set the instrument to complete remains the same (e.g. that you only have the option of setting Data_entry to "Complete" if Required_elements_completed='Y'

#### Link(s) to related issue(s)

* Resolves #6746
